haystack/modeling/__init__.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+try:
+    import torch
+except (ModuleNotFoundError, ImportError) as iexc:
+    raise ImportError("torch not installed, haystack.modeling won't work.")
```

Comment:
Would be good for the user if we explain here how to fix that. Currently, that would be `pip install transformers[torch]`

Reply:
Ah right, good point! Technically this import should appear only if someone tries a direct import from the modeling module, which is rare. But I agree this needs improvement and I forgot about it completely by the time I finished this PR :smile: Thanks!